spring-boot-project/spring-boot-docs/src/main/asciidoc/getting-started.adoc

```diff
@@ -602,64 +601,64 @@ file named `src/main/java/Example.java`:
 
-TIP: The `@RestController` and `@RequestMapping` annotations are Spring MVC annotations
-(they are not specific to Spring Boot). See the {spring-reference}web.html#mvc[MVC section] in
+TIP: The `@RestController` and `@RequestMapping` annotations are Spring MVC annotations.
+(They are not specific to Spring Boot.) See the {spring-reference}web.html#mvc[MVC section] in
 the Spring Reference Documentation for more details.
 
 
 
 [[getting-started-first-application-auto-configuration]]
-==== The @EnableAutoConfiguration annotation
+==== The @EnableAutoConfiguration Annotation
 The second class-level annotation is `@EnableAutoConfiguration`. This annotation tells
-Spring Boot to "`guess`" how you will want to configure Spring, based on the jar
+Spring Boot to "`guess`" how you want to configure Spring, based on the jar
 dependencies that you have added. Since `spring-boot-starter-web` added Tomcat and
-Spring MVC, the auto-configuration will assume that you are developing a web application
-and setup Spring accordingly.
+Spring MVC, the auto-configuration assumes that you are developing a web application
+and sets up Spring accordingly.
 
 .Starters and Auto-Configuration
 ****
 Auto-configuration is designed to work well with "`Starters`", but the two concepts
 are not directly tied. You are free to pick-and-choose jar dependencies outside of the
-starters and Spring Boot will still do its best to auto-configure your application.
+starters and Spring Boot still does its best to auto-configure your application.
 ****
 
 
 
 [[getting-started-first-application-main-method]]
-==== The "`main`" method
+==== The "`main`" Method
 The final part of our application is the `main` method. This is just a standard method
 that follows the Java convention for an application entry point. Our main method delegates
-to Spring Boot's `SpringApplication` class by calling `run`. `SpringApplication` will
-bootstrap our application, starting Spring which will in turn start the auto-configured
+to Spring Boot's `SpringApplication` class by calling `run`. `SpringApplication`
+bootstraps our application, starting Spring, which, in turn, starts the auto-configured
 Tomcat web server. We need to pass `Example.class` as an argument to the `run` method to
 tell `SpringApplication` which is the primary Spring component. The `args` array is also
 passed through to expose any command-line arguments.
 
 
 
 [[getting-started-first-application-run]]
-=== Running the example
-At this point our application should work. Since we have used the
-`spring-boot-starter-parent` POM we have a useful `run` goal that we can use to start
+=== Running the Example
+At this point, your application should work. Since you used the
+`spring-boot-starter-parent` POM, you have a useful `run` goal that you can use to start
 the application. Type `mvn spring-boot:run` from the root project directory to start the
-application:
+application. You should output similar to the following:
```

Comment:
Missing "see"? You should output -> You should see output

Reply:
Oops. That's why I need reviewers as much as anyone else does. Thanks for catching it.